style.css

```diff
@@ -720,17 +720,22 @@ footer {
   color: #6070ff;
 }
 
```

Comment:
- [ ] Blossom the portfolio is showing the same popup window again and again please make sure to fix it and do the requested changes that I told you about 
![image](https://user-images.githubusercontent.com/74744787/180625526-dd2bba55-3626-4d4a-9b67-a66c64831295.png)